doc/contributing.md

```diff
@@ -485,8 +485,8 @@ For consistency, we also use UNIX-style line endings (`\n`) and file permission
 644 (`-rw-r--r--`) throughout the whole project.
 Don't worry if you forget to do it. Our continuous integration systems will
 warn us and you can make a new commit with the formatted code.
-Even better, you can just write `/format` in the first line of any comment in a
-pull request to lint the code automatically.
+Even better, you can just write `/format` and/or `pre-commit.ci autofix` in the first
+line of any comment in a pull request to lint the code automatically.
```

Comment:
I'm thinking of whether to just add `pre-commit run --all-files` to the Makefile, or have it listed separately. Main consideration is that Windows users won't have `make` installed by default, so they would only be able to run `pre-commit run --all-files`.

Reply:
`make` is added as a development dependency in `environment.yml`, so even Windows developers have `make` installed.

Reply:
Oh yes, forgot that it is possible to conda install make now! I'll add it to the Makefile then.